src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/GenerateClaimFormForSpecCallbackHandler.java

```diff
@@ -49,6 +50,9 @@ public class GenerateClaimFormForSpecCallbackHandler extends CallbackHandler {
     private final LitigantInPersonFormGenerator litigantInPersonFormGenerator;
     private final FeatureToggleService toggleService;
 
+    @Value("${stitching.enabled}")
+    private boolean stitchEnabled;
```

Comment:
just confirming, are we covering both scenarios n our tests, with this switch on and off?